doc/cabal-package.rst

```diff
@@ -1119,8 +1154,6 @@ Examples:
     Outdated dependencies:
     HUnit ==1.3.1.1 (latest: 1.3.1.2)
 
-See `the command documentation <cabal-commands.html#cabal-outdated>`__ for a
-list of available flags.
```

Comment:
Why was this deleted? If you inlined it above, did it not exist before? If it existed before, then this is an extraneous change and should be reverted

Reply:
doc/cabal-package.rst

subsequent to my typo correction

the upstream file was changed

could not rebase so reverted that change

still could not rebase

so used upstream one